Tools/MiniBrowser/mac/WK2BrowserWindowController.m

```diff
@@ -143,6 +146,13 @@ - (void)awakeFromNib
     };
 
     _zoomTextOnly = NO;
+
+    _pointerLockBanner = [[CATextLayer alloc] init];
+    [_pointerLockBanner setString:[[NSAttributedString alloc] initWithString:@"Your mouse pointer is hidden. Press Esc (Escape) once to reveal your mouse pointer." attributes:@{ NSFontAttributeName:[NSFont systemFontOfSize:16], NSForegroundColorAttributeName:NSColor.blackColor }]];
+    [_pointerLockBanner setBackgroundColor:NSColor.lightGrayColor.CGColor];
+    [_pointerLockBanner setWrapped:YES];
+    [_pointerLockBanner setFrame:CGRectMake(0, 0, 0, testHeaderBannerHeight)];
+    [_pointerLockBanner setContentsScale:[NSScreen mainScreen].backingScaleFactor];
```

Comment:
I would have lazily created this in `_webViewDidRequestPointerLock` – but fine if you want to land as-is.

`if (! _pointerLockBanner) ...`

Reply:
Good idea! Adopted in [29836c1](https://github.com/WebKit/WebKit/pull/19617/commits/29836c1f8dc61f8925098a09437e60e0e95b6403)